src/main/kotlin/de/smartsquare/starter/mqttadmin/client/ClientConfiguration.kt

```diff
@@ -4,8 +4,8 @@ import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 
 @Configuration
-open class ClientConfiguration {
+class ClientConfiguration {
```

Comment:
Not sure if `open` can be safely removed here, since it is needed when declaring as a Spring bean.